build-tools/src/testFixtures/groovy/org/elasticsearch/gradle/fixtures/AbstractGradleFuncTest.groovy

```diff
@@ -33,14 +34,16 @@ abstract class AbstractGradleFuncTest extends Specification {
     File propertiesFile
     File projectDir
 
+    public boolean configurationCacheCompatible = false
```

Comment:
This adds support for testing our plugins with configuration-cache. at one point we can hopefully make the switch to `true` by default.